microservices/payment/models/Teacher.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Float, Integer
+
+from .base import Base
+
+
+class Teacher(Base):
+    __tablename__ = 'teachers'
+
+    id = Column(Integer, primary_key=True)
+    week_profit = Column(Float)
```

Comment:
why decided to have weekly profit

Reply:
This is the teacher's profit for current week. At the end of the week we will pay them their week profit

Reply:
alright then